GLOSSARY.md

```diff
@@ -1,6 +0,0 @@
-# Glossary
-
-## terminus
-
-A command-line tool for interacting with a
-[Pantheon](https://pantheon.io)-hosted site.
```

Comment:
Does this still highlight the terms without the GLOSSARY.md?

Reply:
I don't know. I didn't even know that's what this file was for, I didn't see it linked anywhere. But, we are starting a big documentation overhaul, and have not gotten to the pantheon bits yet. This is baby-step-1 going into a staging branch.

Reply:
Documentation on what GLOSSARY.md does: https://toolchain.gitbook.com/lexicon.html